test/integration/death_handling_test.go

```diff
@@ -64,10 +65,10 @@ func testDie(t *testing.T, killSrc bool, nodesCount int) {
 	k8s.Prepare("nsmd", "nsc", "nsmd-dataplane", "icmp-responder-nse")
 	logrus.Printf("Cleanup done: %v", time.Since(s1))
 
-	nodes := nsmd_test_utils.SetupNodes(k8s, nodesCount)
+	nodes := nsmd_test_utils.SetupNodes(k8s, nodesCount, defaultTimeout)
```

Comment:
should we put defaultTimeout inside? for test to be simplier?

Reply:
problem is `SetupNodes` is in another package, while `defaultTimeout` is defined here `const.go` to be able to have a central place to tweak timeouts.

Reply:
ok